apps/platform_runner/mix.exs

```diff
@@ -24,19 +24,21 @@ defmodule PlatformRunner.MixProject do
 
   defp deps do
     [
+      {:benchee, "~> 1.0", only: [:test, :integration]},
+      {:bypass, "~> 1.0", only: [:test, :integration]},
       {:cowlib, "~> 2.8.0", override: true},
-      {:service_receive, in_umbrella: true},
-      {:service_gather, in_umbrella: true},
-      {:service_broadcast, in_umbrella: true},
-      {:service_persist, in_umbrella: true},
+      {:credo, "~> 1.3", only: [:dev]},
+      {:divo, "~> 1.1"},
       {:service_acquire, in_umbrella: true},
+      {:service_aggregate, in_umbrella: true},
+      {:service_broadcast, in_umbrella: true},
+      {:service_gather, in_umbrella: true},
       {:service_orchestrate, in_umbrella: true},
-      {:divo, "~> 1.1"},
-      {:websockex, "~> 0.4.0"},
-      {:bypass, "~> 1.0", only: [:test, :integration]},
-      {:benchee, "~> 1.0", only: [:test, :integration]},
-      {:credo, "~> 1.3", only: [:dev]},
-      {:testing, in_umbrella: true, only: [:test, :integration]}
+      {:service_persist, in_umbrella: true},
+      {:service_profile, in_umbrella: true},
+      {:service_receive, in_umbrella: true},
+      {:testing, in_umbrella: true, only: [:dev, :test, :integration]},
```

Comment:
Not sure if we made the correct change here but we were attempting to satisfy this error when running `mix deps.get`:

```
Dependencies have diverged:
* testing (../testing)
  the :only option for dependency testing

  > In mix.exs:
    {:testing, [env: :dev, path: "../testing", in_umbrella: true, only: [:test, :integration]]}

  does not match the :only option calculated for

  > In /Users/rick.ucker/code/hindsight/apps/service_aggregate/mix.exs:
    {:testing, [env: :dev, path: "../testing", in_umbrella: true, only: [:dev, :test]]}

  Ensure you specify at least the same environments in :only in your dep
** (Mix) Can't continue due to errors on dependencies
```